pycode/memilio-epidata/memilio/epidata/defaultDict.py

```diff
@@ -51,7 +51,8 @@
     'file_format': 'json_timeasstring',
     'no_raw': False,
     'rep_date': False,
-    'sanitize_data': 1
+    'sanitize_data': 1,
+    'to_dataset': False
```

Comment:
Since this does not really change the functionality of the downloads, please add this to the download_config file instead of the default_dict.

Reply:
Makes sense. I will move it to download_config